janitor/utils.py

```diff
@@ -31,11 +34,14 @@ def check(varname: str, value, expected_types: list):
     """
     is_expected_type = False
     for t in expected_types:
-        if isinstance(value, t):
-            is_expected_type = True
+        if t is callable:
+            is_expected_type = t(value)
+        else:
+            is_expected_type = isinstance(value, t)
+        if is_expected_type is True:
             break
 
-    if not is_expected_type:
+    if is_expected_type is False:
```

Comment:
I'm not sure where I saw the following point, but I think the idiomatic/preferred way of writing conditionals based on whether something is boolean True is to use the `if is_expected_type` or `if not is_expected_type` pattern.